packages/tabsheet/src/styles/vaadin-tabsheet-base-styles.js

```diff
@@ -42,7 +42,7 @@ export const tabSheetStyles = [
       position: relative;
       flex: 1;
       box-sizing: border-box;
-      padding: var(--vaadin-tabsheet-padding, var(--vaadin-padding-s));
+      padding: var(--vaadin-tabsheet-padding, var(--vaadin-padding-l));
```

Comment:
This would make the content / prefix look a bit misaligned, which is especially noticeable in the `no-border` screenshot. Not a blocker since padding is easily configurable with a custom property anyway.

Reply:
Yeah, I contemplated on this, and it was a very on-the-spot judgement call to use a bigger padding on the content. Let’s see if it holds up with Aura, or if we want to adjust it.